llvm/cmake/modules/HandleLLVMOptions.cmake

```diff
@@ -1217,6 +1217,15 @@ if (LLVM_BUILD_INSTRUMENTED)
         CMAKE_EXE_LINKER_FLAGS
         CMAKE_SHARED_LINKER_FLAGS)
     endif()
+  elseif(uppercase_LLVM_BUILD_INSTRUMENTED STREQUAL "CSSPGO")
+    append("-fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fno-optimize-sibling-calls -fpseudo-probe-for-profiling"
```

Comment:
I think we also need `-fdebug-info-for-profiling`. 

we need to check if compiler (CMAKE_CXX_COMPILER_ID) is clang here because pseudo probe is only for clang.